client/src/components/resources/CellLine/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
-import { SearchResult, SearchResultDetail } from '../../components/SearchResult'
-import DetailsTable from '../../components/DetailsTable'
-import { HeaderRow } from '../../components/HeaderRow'
+import { SearchResult, SearchResultDetail } from '../../SearchResult'
+import DetailsTable from '../../DetailsTable'
+import { HeaderRow } from '../../HeaderRow'
```

Comment:
Just wondering, but does javascript allow absolute imports?

Reply:
Honestly I don't know what the default functinality is... the code and this part especially is rewritten by babel and webpack I think there is a way to configure mappings for paths so we could say `import thing from components` and it would know to look for `thing` in `components/thing/index.js` but I haven't had a chance to dig into that yet